hbase-it/src/test/java/org/apache/hadoop/hbase/HBaseClusterManager.java

```diff
@@ -217,7 +217,7 @@ protected String findPidCommand(ServiceType service) {
     }
 
     public String signalCommand(ServiceType service, String signal) {
-      return String.format("%s | xargs sudo kill -s %s", findPidCommand(service), signal);
+      return String.format("%s | xargs kill -s %s", findPidCommand(service), signal);
```

Comment:
Why remove sudo?

Reply:
find pid command using ps ux which lists the processes owned by the user who is running the command. So to kill user owned processes sudo is not required hence removed.